packages/did/src/DidDetails/FullDidDetails.utils.ts

```diff
@@ -28,10 +29,24 @@ const methodMapping: Record<string, VerificationKeyRelationship | undefined> = {
   web3Names: 'authentication',
 }
 
-export function getKeyRelationshipForExtrinsic(
-  extrinsic: Extrinsic
+function getKeyRelationshipForCall<A extends AnyTuple = AnyTuple>(
```

Comment:
Since the polkadot uses name `method` for that, I’d prefer this function to be called `getKeyRelationshipForMethod`. 

I’d also prefer to drop the old one completely.

Finally, single-letter types are the new single-letter variables. Could we replace `A` with a sensible name?

Reply:
> Since the polkadot uses name method for that, I’d prefer this function to be called getKeyRelationshipForMethod.

It expects a `CallBase` which extends the `IMethod`. So i guess both are valid. I'm not sure where the `Method` naming comes from since I never saw that naming on the chain side. There it's always a `Call`. 🤷 

> I’d also prefer to drop the old one completely.

You refer to `getKeyRelationshipForExtrinsic`? I would agree with Timo that we should keep it consistent. If all our calls expect `Extrinsics` this one should too.

Reply:
> You refer to getKeyRelationshipForExtrinsic? I would agree with Timo that we should keep it consistent. If all our calls expect Extrinsics this one should too.

I mean as an application developer I would be confused about which method to use: getKeyRelationshipForExtrinsic or getKeyRelationshipForMethod, if both are available. This is my preference though, so if you guys both think it should remain, let’s keep it.

Reply:
> as an application developer I would be confused

You are right about that. Therefore one of them is exported while the other is not. The `getKeyRelationshipForMethod` is only there because I call it recursively for batch calls. And a batch call doesn't contain `Extrinsic`s but `CallBase`s